src/index.js

```diff
@@ -8,41 +8,42 @@ const feedWorker = require('./feed-worker');
 
 /**
  * Process a string into a list of Objects, each with a feed URL
- * @param {String} lines 
+ * @param {String} lines
  */
 function processFeedUrls(lines) {
   return lines
     // Divide the file into separate lines
     .split(/\r?\n/)
-    // Basic filtering to remove any ines that don't look like a feed URL 
-    .filter(line => line.startsWith('http'))
+    // Basic filtering to remove any ines that don't look like a feed URL
+    .filter((line) => line.startsWith('http'))
     // Convert this into an Object of the form expected by our queue
-    .map(url => ({ url }));
+    .map((url) => ({ url }));
 }
 
 /**
  * Adds feed URL jobs to the feed queue for processing
- * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed 
+ * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed
  */
-async function enqueueFeedJobs(feedJobs) {
-  for(let feedJob of feedJobs) {
+function enqueueFeedJobs(feedJobs) {
```

Comment:
Does this not also need to be `async`, since we are `await`ing within the loop?

Reply:
Well, the loop is in another function that is async, where it's then awaited. I don't know if each function that uses that function needs to be async though. It does work either way.

Reply:
I think it worked without the async at the top simply because there was nothing after to run asynchronously.

Reply:
If you `await` in a function, then you are necessarily in an `async` function, and need to decorate your function with the `async` keyword.  So yes, adding this back would be good.

Reply:
It is added back already.